packages/pn-personagiuridica-webapp/src/components/Contacts/AddSpecialContactDialog.tsx

```diff
@@ -62,7 +62,7 @@ const AddSpecialContactDialog: React.FC<Props> = ({
     checkIfSenderIsAlreadyAdded(formik.values.sender, e.target.value as ChannelType);
   };
 
-  const checkIfSenderIsAlreadyAdded = (sender: Party, channelType: ChannelType) => {
+  const checkIfSenderIsAlreadyAdded = (sender: Party, channelType: ChannelType | '') => {
```

Comment:
remove the | '' and put an if check in addressTypeChangeHandler and senderChangeHandler

Reply:
done